### PR TITLE
Update sdk ref

### PR DESCRIPTION
**Pull request by bthaile**
thanks @Justbeingjustin for the assist. 

UI to use top level references. 

@adrake33  add AugurSDK to window.

### CODE REVIEW

File: packages/augur-ui/src/modules/events/actions/log-handlers.ts [hunk 1]
```diff
@@ -39,10 +39,9 @@ import { AppState } from 'store';
 import { updateBlockchain } from 'modules/app/actions/update-blockchain';
 import { isSameAddress } from 'utils/isSameAddress';
 import {
-  OrderEventType,
-  ParsedOrderEventLog,
-} from '@augurproject/sdk/build/state/logs/types';
-import { TXStatus } from '@augurproject/sdk/build/events';
+  Logs
+} from '@augurproject/sdk';
+import { Events } from '@augurproject/sdk';
```
Comment (adrake33) [inline packages/augur-ui/src/modules/events/actions/log-handlers.ts]: Same here.

File: packages/augur-ui/src/modules/events/actions/add-update-transaction.ts [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
 import { PUBLICTRADE, CANCELORDER, TX_ORDER_ID, TX_MARKET_ID, TX_TRADE_GROUP_ID } from 'modules/common/constants';
 import { UIOrder } from 'modules/types';
 import { convertTransactionOrderToUIOrder } from './transaction-conversions';
 import { addPendingOrder, updatePendingOrderStatus, removePendingOrder } from 'modules/orders/actions/pending-orders-management';
 import { ThunkDispatch } from 'redux-thunk';
 import { Action } from 'redux';
 import { AppState } from 'store';
-import { MarketInfo } from '@augurproject/sdk/build/state/getter/Markets';
-import { TXEventName } from '@augurproject/sdk/build';
+import { Getters, TXEventName } from '@augurproject/sdk';
```
Comment (adrake33) [inline packages/augur-ui/src/modules/events/actions/add-update-transaction.ts]: Nit: Could combine the 2 @augurproject/sdk imports here into a single line.

File: packages/augur-ui/src/modules/events/actions/transaction-conversions.ts [hunk 3]
```diff
@@ -1,10 +1,10 @@
 import { UIOrder } from 'modules/types';
-import { convertOnChainPriceToDisplayPrice, convertOnChainAmountToDisplayAmount, numTicksToTickSize, QUINTILLION } from '@augurproject/sdk/build';
+import { convertOnChainPriceToDisplayPrice, convertOnChainAmountToDisplayAmount, numTicksToTickSize, QUINTILLION } from '@augurproject/sdk';
 import { SELL, BUY, TX_TRADE_GROUP_ID, TX_MARKET_ID, TX_OUTCOME_ID, TX_PRICE, TX_AMOUNT, TX_DIRECTION } from 'modules/common/constants';
-import { MarketInfo } from '@augurproject/sdk/build/state/getter/Markets';
 import { createBigNumber } from 'utils/create-big-number';
+import { Getters } from "@augurproject/sdk";
```
Comment (adrake33) [inline packages/augur-ui/src/modules/events/actions/transaction-conversions.ts]: And here.

File: packages/augur-ui/src/modules/trades/actions/update-trade-cost-shares.ts [hunk 4]
```diff
@@ -7,8 +7,8 @@ import { ThunkDispatch } from 'redux-thunk';
 import { Action } from 'redux';
 import { NodeStyleCallback } from 'modules/types';
 import { simulateTrade, simulateTradeGasLimit } from 'modules/contracts/actions/contractCalls';
-import { SimulateTradeData } from '@augurproject/sdk/build';
-import { MarketInfo } from '@augurproject/sdk/build/state/getter/Markets';
+import { SimulateTradeData } from '@augurproject/sdk';
+import { Getters } from '@augurproject/sdk';
```
Comment (adrake33) [inline packages/augur-ui/src/modules/trades/actions/update-trade-cost-shares.ts]: And here.

**Review by adrake33 — APPROVED** [review]
Lgtm, just recommend consolidating a few of the imports where possible.

**Comment by bthaile** [comment]
Not sure why unit tests are timing out.

image


passes if only Trade.test.ts is run. this is all locally

**Pull request merged**